docs-asciidoc/Makefile

```diff
@@ -35,6 +35,8 @@ html-author-mode:
 apidocs:
 	cd ${root_dir} && sbt java-support/doc
 	cp -r ${root_dir}/java-support/target/api ${site_dir}/docs/current/java/api
+	cd ${root_dir}/node-support && npm i
+	npm i -g jsdoc
```

Comment:
Does it need to be installed globally as well?

Reply:
yea probably not necessary.